lib/facter/package_provider.rb

```diff
@@ -23,3 +28,16 @@
     end
   end
 end
+
+Facter.add(:package_provider) do
+
+  confine do
+    Facter.value(:kernel) == 'windows'
+  end
```

Comment:
And then give this a higher weight (like 100) and confine it to windows.
```suggestion
  has_weight 100
  confine kernel: 'windows'
```